### PR TITLE
[12.0][ADD] website_sale_attribute_filter_price

### PR DESCRIPTION
**Pull request by Tardo**
pricefilter

cc @tecnativa TT23634

### CODE REVIEW

File: website_sale_attribute_filter_price/controllers/website_sale.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        try:
+            custom_max_price = float(post.get("max_price"))
+        except (ValueError, TypeError):
+            custom_max_price = max_price
+        if custom_min_price > custom_max_price:
+            custom_max_price, custom_max_price = custom_min_price, custom_max_price
+        # Call Super with context
+        request.env.context = dict(request.env.context,
+            price_vals = [custom_min_price, custom_max_price]
+        )
```
Comment (chienandalu) [inline website_sale_attribute_filter_price/controllers/website_sale.py]: The lint error comes from here
```suggestion
        request.env.context = dict(
            request.env.context,
            price_vals=[custom_min_price, custom_max_price])
```

**Review by chienandalu — COMMENTED** [review]
* Check travis
* There's some issue loading the lib assets:
image

**Comment by pedrobaeza** [comment]
Isn't there a slider in existing resources for not requiring an extra library?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Two functional problems:

- Incorrect limits:

Selección_009

The same may happen when you search for a text, and also applies to the minimum value.

- No price filter:

Selección_010

And check also that Travis is red

**Review by chienandalu — COMMENTED** [review]
A pair of usability problems I found

price-filter

**Comment by Tardo** [comment]
@chienandalu thanks. Improved the usability... but the problem with the 'Sort By' is a 'core' issue. This happens because the inputs can't use 'keep' method and the form doesn't have the 'order' value (as "input type='hidden'")

**Comment by chienandalu** [comment]
Great. Being picky, one would expect to re-apply the filter once you click on the reset button
erase-filter

**Comment by Tardo** [comment]
@chienandalu changes made, removed all custom css and compatible with currency symbol position

pricefilter

**Review by chienandalu — APPROVED** [review]
Great!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**